src/utils.ts

```diff
@@ -6,3 +6,5 @@ export const getStatusByDuration = (startDate: string, endDate: string): Status
   if (moment().isAfter(endDate)) return 'closed'
   return 'active'
 }
+
+export const normalizeString = (text: string): string => text.trim().toLowerCase()
```

Comment:
Please, add a test suite for this function, if you want to improve code quality.